modules/launcher/src/main/java/org/mule/module/launcher/MuleApplicationClassLoader.java

```diff
@@ -46,6 +49,8 @@
 
     protected static final URL[] CLASSPATH_EMPTY = new URL[0];
 
+    private static final String DEFAULT_LEAK_CLEANER_CLASS = "DefaultLeakCleaner.class";
```

Comment:
Why? Can't you either specify implementation directly or if that doen't make sense use Service Provider approach.